apps/e2e/cypress/e2e/calls.cy.ts

```diff
@@ -126,7 +126,7 @@ context('Calls tests', () => {
 
   // TODO: Maybe this should be moved to another file called permissions because its testing more call permissions than calls.
   it('A user should not be able to see/visit calls', () => {
-    cy.login('user1');
+    cy.login('user1', 1);
```

Comment:
I was wondering if it is possible to use the `initialDBData` roles here as it will be more self-explanatory what the second argument means and much easier to maintain in the end.